oat-sa/extension-tao-itemqti
# fix: correct table th line for FF in item and test preview

Related to: LSI-864
Lines in match interaction not visible in item and test preview in Firefox:


How to test:
open your tao instance in firefox browser
Create an item and add match interaction
preview item
add item to test: preview test

Expected behavior:
Lines are visible in ` th`  of interaction table for both cases:
![image](https://user-images.githubusercontent.com/60346520/183899802-c16d7d0e-9933-44d9-89dc-870c4d2118fc.png)
